File: net/merge/Merger_pdf_pptx.cs
namespace aspose_snippets.net
{
    public static partial class Merger
    {
        public static void pdf_to_pptx()
        {
            var pathSource1 = @"..\..\TestData\test.pdf";
            var pathSource2 = @"..\..\TestData\Second\test.pdf";

            //read pdf file to Aspose Document
            var firstDoc = new Aspose.Pdf.Document(pathSource1);
            var secondDoc = new Aspose.Pdf.Document(pathSource2);

            var outputDoc = new Aspose.Pdf.Document
            {
                //set less memory usage with unload instead of fast performance
                EnableObjectUnload = true
            };

            foreach (var page in firstDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            foreach (var page in secondDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            //save pdf to Microsoft PowerPoint
            outputDoc.Save("test.pptx",
                            new Aspose.Pdf.PptxSaveOptions
                            {
                                //save all content on page as single image
                                SlidesAsImages = true
                            });
        }
    }
}

File: java/merge/src/aspose_snippets/java/Merger_pdf_pptx.java
package aspose_snippets.java;

public class Merger_pdf_pptx {
    public static void Execute() {
        String pathSource1 = "../../TestData/test.pdf";
        String pathSource2 = "../../TestData/Second/test.pdf";

        //read pdf file to Aspose Document
        com.aspose.pdf.Document firstDoc = new com.aspose.pdf.Document(pathSource1);
        com.aspose.pdf.Document secondDoc = new com.aspose.pdf.Document(pathSource2);

        com.aspose.pdf.Document outputDoc = new com.aspose.pdf.Document();
        //set less memory usage with unload instead of fast performance
        outputDoc.setEnableObjectUnload(true);

        for (com.aspose.pdf.Page page : firstDoc.getPages()) {
            //add page from one document to another directly
            outputDoc.getPages().add(page);
        }

        for (com.aspose.pdf.Page page : secondDoc.getPages()) {
            //add page from one document to another directly
            outputDoc.getPages().add(page);
        }

        com.aspose.pdf.PptxSaveOptions opt1 = new com.aspose.pdf.PptxSaveOptions();
        //save all content on page as single image
        opt1.setSlidesAsImages(true);
        outputDoc.save("Merger_pdf_pptx.pptx", opt1);
    }
}
